### ISSUE TITLE
Unable to Save Default Checklist Items

### ISSUE THREAD

**Issue by jwrubel45** [mohitbagra/vsts-extensions#27]
Hello,

I am unable to save checklist items, either manually in the work item type or by setting a default through the work item checklist. What am I doing wrong?

Thanks,

Jon

**Comment by mohitbagra** [comment]
What sort of error are you getting? Can you share a screenshot or a gif/video with the repro?

**Comment by jwrubel45** [comment]
Hi, thanks for the response. I'm actually not getting an error prompt. When I select the work item checklists, I see the work item types running down the left hand side of the screen, and the page seems to be letting me add new items to the checklist for the work item type I select. However, when I type one in, and click the blue check mark, nothing happens. In the tutorial from MSFT, it seems like a list should be generated, but it doesn't seem to be working. I've attached a couple of screenshots. 
<img width="617" alt="checklist 1" src="https://user-images.githubusercontent.com/40494737/41874063-cbaecca8-7894-11e8-8fad-086b6c089263.PNG">
<img width="637" alt="checklist 2" src="https://user-images.githubusercontent.com/40494737/41874074-d0ba8534-7894-11e8-887a-a21823078677.PNG">

**Comment by jwrubel45** [comment]
You'll notice that when I select the work item type "ticket", you see the prompt "no default checklist items crated for "ticket", but you don't see that same prompt for the user story. This is because I've entered items, but I just can't see them.

**Comment by mohitbagra** [comment]
I am able to repro this only on Internet Explorer. Can you check if you reproduce this error on any other browser (Chrome, firefox, Edge)?

**Comment by jwrubel45** [comment]
you are correct. It looks like it's working great in Firefox. Apologies for not thinking of that before messaging you, but I'm glad we figured it out! I'll make sure our team knows to use Firefox instead of explorer. Any thoughts on whether you'll fix it for Explorer though in case people forget?

**Comment by mohitbagra** [comment]
I created a separate issue for that (#28 ). I am closing this one since you are unblocked now.
Meanwhile I'll try to fix this for IE when I get some time. Thanks for reporting this. IE is my 4th favorite browser so sometimes I forget testing in it :(
